Lib/test/test_dis.py

```diff
@@ -360,6 +367,17 @@ def test_boundaries(self):
         self.assertEqual(dis.opmap["EXTENDED_ARG"], dis.EXTENDED_ARG)
         self.assertEqual(dis.opmap["STORE_NAME"], dis.HAVE_ARGUMENT)
 
+    def test_widths(self):
+        for opcode, opname in enumerate(dis.opname):
+            if opname in ('BUILD_MAP_UNPACK_WITH_CALL',
+                          'BUILD_TUPLE_UNPACK_WITH_CALL'):
```

Comment:
While I don't think it should block this change (since it's an existing problem) I wonder if it might it be worth special-casing these for display purposes using something like:

    BUILD_MAP_UNPACK_WITH_CALL
    ...MAP_UNPACK+CALL
    12345678901234567890

    BUILD_TUPLE_UNPACK_WITH_CALL
    ...TUPLE_UNPACK+CALL
    12345678901234567890